Update full tutorials from develop (#567)
Avaiga/taipy-doc: docs/tutorials/complete_applications/scenario_management_overview/index.md

!!! important "Supported Python versions"

    Taipy requires **Python 3.8** or newer.

Welcome to the **Tutorial** guide for scenario management.

> Taipy facilitates pipeline orchestration. There are a lot of reasons for using Taipy backend:

Reviewer: ```suggestion
Taipy facilitates pipeline orchestration. There are several good reasons to use the Taipy backend:
```

- Taipy efficiently manages the execution of your functions/pipelines.

- Taipy manages data sources and monitors KPIs.

> - Taipy provides easy management of multiple pipelines and end-user scenarios, which comes in handy in the context of Machine Learning or Mathematical optimization.

Reviewer: ```suggestion
- Taipy makes it simple to oversee multiple pipelines and end-user scenarios, especially useful in Machine Learning or Mathematical optimization contexts.
```
